apps/docs/content/guides/realtime/getting_started.mdx

```diff
@@ -0,0 +1,710 @@
+---
+id: 'getting-started'
+title: 'Getting Started with Realtime'
+description: 'Learn how to build real-time applications with Supabase Realtime'
+subtitle: 'Learn how to build real-time applications with Supabase Realtime'
+sidebar_label: 'Getting Started'
+---
+
+## Quick start
+
+### 1. Install the client library
+
+<Tabs
+  scrollable
+  size="small"
+  type="underlined"
+  defaultActiveId="js"
+  queryGroup="language"
+>
+<TabPanel id="js" label="JavaScript">
+
+```bash
+npm install @supabase/supabase-js
+```
+
+</TabPanel>
+<TabPanel id="dart" label="Flutter">
+
+```bash
+flutter pub add supabase_flutter
+```
+
+</TabPanel>
+<TabPanel id="swift" label="Swift">
+
+```swift
+let package = Package(
+    // ...
+    dependencies: [
+        // ...
+        .package(
+            url: "https://github.com/supabase/supabase-swift.git",
+            from: "2.0.0"
+        ),
+    ],
+    targets: [
+        .target(
+            name: "YourTargetName",
+            dependencies: [
+                .product(
+                    name: "Supabase",
+                    package: "supabase-swift"
+                ),
+            ]
+        )
+    ]
+)
+```
+
+</TabPanel>
+<TabPanel id="python" label="Python - PIP">
+
+```bash
+pip install supabase
+```
+
+</TabPanel>
+<TabPanel id="python" label="Python - Conda">
+
+```bash
+conda install -c conda-forge supabase
+```
+
+</TabPanel>
+</Tabs>
+
+### 3. Create your first Channel
+
+Channels are the foundation of Realtime. Think of them as rooms where clients can communicate. Each channel is identified by a topic name and if they are public or private.
+
+<Tabs
+  scrollable
+  size="small"
+  type="underlined"
+  defaultActiveId="js"
+  queryGroup="language"
+>
+<TabPanel id="js" label="JavaScript">
+
+```ts
+// Create a channel with a descriptive topic name
+const channel = supabase.channel('room:lobby:messages', {
+  config: { private: true }, // Recommended for production
+})
+```
+
+</TabPanel>
+<TabPanel id="dart" label="Flutter">
+
+```dart
+// Create a channel with a descriptive topic name
+final channel = supabase.channel('room:lobby:messages');
+```
+
+</TabPanel>
+<TabPanel id="swift" label="Swift">
+
+```swift
+// Create a channel with a descriptive topic name
+let channel = supabase.realtimeV2.channel("room:lobby:messages")
+```
+
+</TabPanel>
+<TabPanel id="python" label="Python">
+
+```python
+# Create a channel with a descriptive topic name
+channel = supabase.channel('room:lobby:messages')
+```
+
+</TabPanel>
+</Tabs>
+### 2. Initialize the client
+
+Get your project URL and key from your [Supabase Dashboard](/dashboard/project/_/settings/api).
```

Comment:
```suggestion
Get your project URL and key.

<$Partial path="api_settings.mdx" />
```

This is something new we're trying to use to standardise key instructions during the transition between anon and publishable.